zuul/connection/github.py

```diff
@@ -220,6 +220,13 @@ def _pull_request_to_event(self, pr_body):
 
         event.title = pr_body.get('title')
 
+        # get the statuses
+        project_name = head.get('repo').get('full_name')
+        head_owner, head_project = project_name.split('/')
+        event.statuses = self.connection.getCommitStatuses(head_owner,
+                                                           head_project,
+                                                           head)
```

Comment:
Also, I've just been poking more closely at the github API and it appears that the commit statuses are attached to the ref in the upstream owner/project and not on the head owner's?  That 

$user proposes a PR /w $sha from $user/$project into $upstream/$project.

The ref at $user/$project:$sha (what is being fetched here) contains no statuses, they are instead attached to the ref at $upstream/$project:$sha